workshops/templates/workshops/admin_dashboard.html

```diff
@@ -39,6 +40,13 @@
     <table class="table table-striped">
     {% for event in current_events %}
     <tr>
+      {% for tag in carpentries %}
+        <td class="text-center" width="25px">
+          {% if tag in event.tags.all %}
+            {% bootstrap_tag tag.name %}
+          {% endif %}
+        </td>
+      {% endfor %}
```

Comment:
Please make sure that this doesn't yield N queries here. I know I got it down to `const` queries per this "current events" column, but I don't remember with which code.